### PR TITLE
Add REST API compute and datastore Doc

### PR DESCRIPTION
**Pull request by kunZooberg**
Add REST API compute and datastore Doc

### CODE REVIEW

**Comment by kunZooberg** [comment]
This is Happy Path Sample of AML compute and datastore.

File: examples/AzureML-REST-API/compute.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+         }
+        }
+
+header = {'Authorization': 'Bearer ' + token, "Content-type": "application/json"}
+create_compute_url = "https://management.azure.com/subscriptions/{}/resourceGroups/{}/providers/Microsoft.MachineLearningServices/workspaces/{}/computes/{}?api-version={}".format(subid, rg, ws, compute_name, api_verison)
+
+resp = requests.put(create_compute_url, headers=header, json=body)
+print resp.text
+```
+
```
Comment (grmadala) [inline examples/AzureML-REST-API/compute.md:144]: Adding an example for 
1. how to poll for status
2. update compute 

will be more helpful

**Review by grmadala — APPROVED** [review]

File: examples/AzureML-REST-API/compute.md (hunk 1 above)
Comment (kunZooberg) [reply examples/AzureML-REST-API/compute.md:144]: Rest operation of resource seems don't have a progress to poll. Is "get method" enough? 
And "compute update" interface is same with "compute create" interface.

Comment (kunZooberg) [reply examples/AzureML-REST-API/compute.md]: And update compute has the same interface with create compute.

Comment (grmadala) [reply examples/AzureML-REST-API/compute.md:144]: including this information in the sample can be helpful -  "compute update" interface is same with "compute create" interface.

Comment (kunZooberg) [reply examples/AzureML-REST-API/compute.md:144]: Sure, make sense. Will add this information.

**Pull request merged**